docs/source/ml.rst

```diff
@@ -105,8 +83,10 @@ Custom Models
 
 You can create your own :class:`~sklearn.pipeline.Pipeline` instead of using one of
 the pre-defined ones. Create the object and pass it as the ``model`` parameter
-of the :class:`~civis.ml.ModelPipeline`. Your model must be built from libraries which CivisML
-recognizes. You can use code from
+of the :class:`~civis.ml.ModelPipeline`. Your model must follow the
+scikit-learn API, and you will need to include any dependencies as
+:ref:`custom-dependencies` if they are not already installed in
+CivisML. Common libraries you may want to use include: 
```

Comment:
"Common libraries you may want to use" --> "Preinstalled libraries available for your use"?